interfaces/Portalicious/src/app/components/button-menu/button-menu.component.ts

```diff
@@ -27,4 +28,17 @@ export class ButtonMenuComponent {
   readonly size = input<'large' | 'small'>();
 
   readonly menuOpen = model(false);
+
+  // Filter out items without command or visible children as these than look like a subheader without any action or children or routerlink
```

Comment:
Filtering out these 'sub header' items without children seems like something generic you would always want. But I am also a bit afraid that the logic in this component is a bit too hidden. 

What do you think?

Reply:
I think it's a good idea, and it makes sense that the component takes care of this logic ✌️